core/Archive/ArchiveInvalidator.php

```diff
@@ -78,11 +78,12 @@ public function getAllRememberToInvalidateArchivedReportsLater()
         // we do not really have to get the value first. we could simply always try to call set() and it would update or
         // insert the record if needed but we do not want to lock the table (especially since there are still some
         // MyISAM installations)
-        $values = Option::getLike($this->rememberArchivedReportIdStart . '%');
+        $values = Option::getLike('%' . $this->rememberArchivedReportIdStart . '%');
```

Comment:
Sometimes I see `'%_' .` and other times `'%' .`, could that be an issue?

Reply:
@diosmosis not really. I just copied behaviour what was done on the right part, to have the same on the left. It'll not make a difference though